client/my-sites/site-settings/form-seo.jsx

```diff
@@ -0,0 +1,379 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import get from 'lodash/get';
+import pickBy from 'lodash/pickBy';
+import isString from 'lodash/isString';
+import page from 'page';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import SectionHeader from 'components/section-header';
+import ExternalLink from 'components/external-link';
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+import notices from 'notices';
+import protectForm from 'lib/mixins/protect-form';
+import FormInput from 'components/forms/form-text-input-with-affixes';
+import FormInputValidation from 'components/forms/form-input-validation';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import CountedTextarea from 'components/forms/counted-textarea';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+const serviceIds = {
+	google: 'google-site-verification',
+	bing: 'msvalidate.01',
+	pinterest: 'p:domain_verify',
+	yandex: 'yandex-verification'
+};
+
+function getGeneralSettingsUrl( slug ) {
+	return `/settings/general/${ slug }`;
+}
+
+function stateForSite( site ) {
+	return {
+		seoMetaDescription: get( site, 'options.seo_meta_description', '' ),
+		verificationServicesCodes: {
+			google: get( site, 'options.verification_services_codes.google', '' ),
+			bing: get( site, 'options.verification_services_codes.bing', '' ),
+			pinterest: get( site, 'options.verification_services_codes.pinterest', '' ),
+			yandex: get( site, 'options.verification_services_codes.yandex', '' )
+		}
+	};
+}
+
+function getMetaTagForService( serviceName = '', content = '' ) {
+	if ( ! content ) {
+		return '';
+	}
+
+	if ( content.indexOf( '<meta' ) !== -1 ) {
+		// We were passed a meta tag already!
+		return content;
+	}
+
+	return `<meta name="${ get( serviceIds, serviceName, '' ) }" content="${ content }" />`;
+}
+
+function isValidVerificationCode( serviceName = '', content = '' ) {
+	if ( content.length === 0 ) {
+		return true;
+	}
+
+	content = getMetaTagForService( serviceName, content );
+
+	return content.indexOf( serviceIds[ serviceName ] ) > -1;
```

Comment:
here's another great spot to use `includes()`

```js
const isValidVerificationCode = ( serviceName = '', content = '' ) =>
	content.length
		? includes( getMetaTagForService( serviceName, content ), serviceIds[ serviceName ] )
		: true;
```

haha, I hope you like _that_ way of writing the function - did it just for you @roundhill